src/components/organisms/SchedulePageModal/SchedulePageModal.tsx

```diff
@@ -1,162 +1,118 @@
-    : currentVenue?.host?.icon;
+      const onWeekdayClick: MouseEventHandler<HTMLLIElement> = (e) => {
+        e.stopPropagation();
+        setSelectedDayIndex(dayIndex);
+      };
 
-  const titleText = hasParentVenue ? parentVenue?.name : currentVenue?.name;
+      return (
+        <li
+          key={day.toISOString()}
+          className={classes}
+          onClick={onWeekdayClick}
+        >
+          {dayIndex === 0 ? "Today" : format(day, "E")}
+        </li>
+      );
+    });
+  }, [selectedDayIndex]);
+
+  const schedule: ScheduleDay = useMemo(() => {
+    const dayStart = addDays(startOfToday(), selectedDayIndex);
+    const daysEvents = relatedVenueEvents
+      .filter(
+        isEventLaterThisDay(selectedDayIndex === 0 ? Date.now() : dayStart)
+      )
+      .map(prepareForSchedule(dayStart, userEventIds));
+
+    const roomNamesInSchedule = new Set(daysEvents.map((event) => event.room));
+
+    const roomsWithEvents = extendRoomsWithDaysEvents(
+      relatedRooms.filter((room) => roomNamesInSchedule.has(room.title)),
+      daysEvents
+    );
+
+    return {
+      isToday: selectedDayIndex === 0,
+      weekday: format(dayStart, "E"),
+      dayStartUtcSeconds: getUnixTime(dayStart),
+      rooms: roomsWithEvents,
+      personalEvents: daysEvents.filter((event) => event.isSaved),
+    };
+  }, [relatedVenueEvents, relatedRooms, userEventIds, selectedDayIndex]);
+
+  const containerClasses = classNames("SchedulePageModal", {
+    "SchedulePageModal--show": isVisible,
+  });
 
-  const subtitleText = hasParentVenue
-    ? parentVenue?.config?.landingPageConfig.subtitle
-    : currentVenue?.config?.landingPageConfig.subtitle;
+  return (
+    <div className={containerClasses}>
+      <ScheduleVenueDescription />
 
-  const descriptionText = hasParentVenue
-    ? parentVenue?.config?.landingPageConfig.description
-    : currentVenue?.config?.landingPageConfig.description;
+      <ul className="SchedulePageModal__weekdays">{weekdays}</ul>
 
-  return (
-    <div>
-      <div className={`schedule-dropdown-body ${isVisible ? "show" : ""}`}>
-        <div className="partyinfo-container">
-          <div className="partyinfo-main">
-            <div
-              className="partyinfo-pic"
-              style={{ backgroundImage: `url(${partyinfoImage})` }}
-            />
-            <div className="partyinfo-title">
-              <h2>{titleText}</h2>
-              <h3>{subtitleText}</h3>
-            </div>
-          </div>
-          <div className="partyinfo-desc">
-            <p>{descriptionText}</p>
-          </div>
-        </div>
-
-        <div className="schedule-container">
-          <ul className="schedule-tabs">{scheduleTabs}</ul>
-          <div className="schedule-day-container">
-            {events}
-            {!hasEvents && (
-              <div>There are no events scheduled for this day.</div>
-            )}
-          </div>
-        </div>
-      </div>
+      {schedule.rooms.length > 0 ? (
```

Comment:
move to a const

Reply:
eg. `hasrooms`

Reply:
In fact.. is this a check that we even need to do here? It seems like the 'no rooms' belongs inside the `Schedule` component itself. It's already being passed `schedule`, so it will have the appropriate data available to it.

Reply:
created the const, moved to Schedule: https://github.com/sparkletown/sparkle/pull/1302/commits/23f86ae3fa959ebb54449aad9c4c96a627741b81